processes_notwindows.go

```diff
@@ -76,7 +76,13 @@ func processesFromProc() ([]ProcStat, error) {
 			log.Errorf("Failed to read cmdline(%s): %s", stats[0], err.Error())
 		}
 
-		stat := ProcStat{PID: pid, Name: string(bytes.TrimRight(comm, "\n")), Cmdline: strings.Replace(string(bytes.TrimRight(cmdline, "\x00")), "\x00", " ", -1)}
+		ppid, err := strconv.Atoi(string(stats[4]))
+
```

Comment:
remove empty line

Reply:
Sometimes I put the empty line before the `if` block. Do you have a link to some convention on golang empty lines? I can't find any mention in the https://github.com/golang/go/wiki/CodeReviewComments

So then I will propose the PR to make all the empty lines consistent.

Reply:
There is not _everything_ mentioned in these guidelines.
I like to group code blocks logically. Function call together with error handling , declaration of variables and also `if` blocks.
Seems this comment was a mistake, the empty line is fine before the if block.
My bad.